terraform/versions.tf

```diff
@@ -1,4 +1,4 @@
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 1.0.3"
```

Comment:
I would put `1.0` here

Reply:
It was to have the last available version but i can change

Reply:
The restriction of the patch version `.3` makes no sense, it's only bug fixes.
What is interesting is to say: you need a version of 1.0 at least.

Reply:
ok i'll change :)